.gitignore

```diff
@@ -3,6 +3,7 @@
 .todo
 coverage/
 dist/
+package-lock.json
```

Comment:
I'm ok with adding a lock file, it's 2018 😄

Reply:
Ups, this shouldn't have been in there. I added a .gitignore entry for this.

Reply:
package-lock.json is gone.

Reply:
Oh I meant that I was ok with adding the `package-lock.json` actually. But that's ok!